### ISSUE TITLE
Mover alterações feitas nesse repo para practice-uffs/website-programa

### ISSUE THREAD

**Issue by Dovyski** [practice-uffs/practice-uffs.github.io#2]
O repositório [practice-uffs/website-programa](http://github.com/practice-uffs/website-programa) já está pronto para receber contribuições. Por favor, migrar qualquer trabalho que tenha sido feito nesse repositório aqui (apenas os arquivos modificados, não tudo) e colocar no novo repositório.

O novo repositório [practice-uffs/website-programa](http://github.com/practice-uffs/website-programa)  já tem os gitignore corretos e um sistema para deploy automático.

Assim que a migração for feita, avisar nesse issue para que eu delete o repo antigo.

**Comment by andrewmsilva** [comment]
Uma pergunta: o arquivo [Gemfile.lock](https://github.com/practice-uffs/website-programa/blob/master/src/Gemfile.lock) não tem que ser incluso no .gitignore também? É que eu reparei que ele é gerado toda vez que roda o `bundler update`, me corrija se eu estiver errado.

**Comment by Dovyski** [comment]
Sim, ele tem que ser incluído. Ele está no gitignore em algum lugar?
